sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_customs_sanitized/view.sql

```diff
@@ -2,6 +2,105 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.docker_fxa_customs_sanitized`
 AS
 SELECT
-  *
+  `date`,
+  logName,
+  (
+    SELECT AS STRUCT
```

Comment:
`SELECT AS STRUCT` is very handy when you want to use `* EXCEPT (...)` or `* REPLACE (...)`, but in cases like these where all the fields are being listed explicitly I'd recommend using the `STRUCT(...)` constructor instead, which should be simpler and more efficient.

Reply:
- You updated some but not all, so now there's an odd mix of the two approaches.
- Unfortunately you can't have a trailing comma after the last `STRUCT()` argument like you can in a `SELECT AS STRUCT` subquery.
- There shouldn't be a space before the opening parenthesis in the `STRUCT()` constructor calls.
- Indention is a bit off within some `STRUCT()` calls.

(re-running `bqetl format` on this file should fix those last two)